ddtrace/internal/processor.py

```diff
@@ -12,11 +13,71 @@
 
 
 @attr.s
-class TraceProcessor(object):
+class Stitcher(object):
```

Comment:
add a docstring?
I got scared to see such a class with a weird name with no doc 😨